webpack/JobWizard/steps/Schedule/index.js

```diff
@@ -106,6 +108,16 @@ const Schedule = ({ scheduleValue, setScheduleValue, setValid }) => {
             }));
           }}
         />
+        <PurposeField
+          isDisabled={repeatType === repeatTypes.noRepeat}
```

Comment:
should we hide the component when `repeatTypes === noRepeat` ?
it's pretty confusing when the field is disabled initially,
another option is to add more explanation on the tooltip

Reply:
It only appears in `»Set up recurring execution«` in the original, so I don't think it has a use out of recurrence

Reply:
> so I don't think it has a use out of recurrence

Correct. Outside of recurring jobs it has no purpose, pun intended :)